src/Command/RecipesCommand.php

```diff
@@ -0,0 +1,256 @@
+            $this->displayPackageInformation(current($recipes));
+
+            return 0;
+        }
+
+        // display a resume of all packages
+        $write = [
+            '',
+            '<bg=blue;fg=white>                      </>',
+            '<bg=blue;fg=white> Recipes available.   </>',
+            '<bg=blue;fg=white>                      </>',
+            '',
+        ];
+
+        /** @var Recipe $recipe */
+        foreach ($recipes as $name => $recipe) {
+            $lockRef = $this->symfonyLock->get($name)['recipe']['ref'] ?? null;
+
+            $additional = '';
+            if (null === $lockRef && null !== $recipe->getRef()) {
+                $additional = '<comment>(recipe not installed)</comment>';
+            } elseif ($recipe->getRef() !== $lockRef) {
+                $additional = '<comment>(update available)</comment>';
+            }
+            $write[] = sprintf(' * %s %s', $name, $additional);
+        }
+
+        $write[] = '';
+        $write[] = '<fg=blue>Run</>:';
+        $write[] = ' * composer symfony:recipes vendor/package to see details about a recipe.';
+        $write[] = ' * composer symfony:recipes:install vendor/package --force -v to update that recipe.';
+        //$write[] = ' * composer symfony:recipes:blame to see a tree of all of the installed/updated files.';
+        $write[] = '';
+
+        $this->getIO()->write($write);
+
+        return 0;
+    }
+
+    private function displayPackageInformation(Recipe $recipe)
+    {
+        $recipeLock = $this->symfonyLock->get($recipe->getName());
+
+        $lockRef = $recipeLock['recipe']['ref'] ?? null;
+        $lockFiles = $recipeLock['files'] ?? null;
+
+        $status = '<comment>up to date</comment>';
+        if ($recipe->isAuto()) {
+            $status = '<comment>auto-generated recipe</comment>';
+        } elseif (null === $lockRef && null !== $recipe->getRef()) {
+            $status = '<comment>recipe not installed</comment>';
+        } elseif ($recipe->getRef() !== $lockRef) {
+            $status = '<comment>update available</comment>';
+        }
+
+        $io = $this->getIO();
+        $io->write('<info>name</info>     : '.$recipe->getName());
+        $io->write('<info>version</info>  : '.$recipeLock['version']);
+        if (!$recipe->isAuto()) {
+            $io->write('<info>repo</info>     : '.sprintf('https://%s/tree/master/%s/%s', $recipeLock['recipe']['repo'], $recipe->getName(), $recipeLock['version']));
+        }
+        $io->write('<info>status</info>   : '.$status);
+
+        if (null !== $lockFiles) {
+            $io->write('<info>files</info>    : ');
+            $io->write('');
+
+            $tree = $this->generateFilesTree($lockFiles);
+
+            $this->displayFilesTree($tree);
+        }
+
+        $io->write([
+            '',
+            'Update this recipe by running composer:',
+            sprintf('<info>symfony:recipes:install %s --force -v</info>', $recipe->getName()),
```

Comment:
So we need to add the alias on #565

Reply:
BTW, as we remove the prefix `symfony', do we keep the support as a name and define an alias? Or we remove directly on the name

Reply:
Yes we should add that alias! I added a comment about it, but I think it got hidden. Should be better now: https://github.com/symfony/flex/pull/565/files#r356801818

> BTW, as we remove the prefix `symfony', do we keep the support as a name and define an alias? Or we remove directly on the name

We should keep the `symfony:` prefix (so this would still be called `symfony:recipes` and the other `symfony:recipes:install`) but both should have aliases (`recipes` and `recipes:install`)

If I'm not answering the correct question, ping me on Slack ;)

Reply:
Yes, I did miss it.